.travis.yml

```diff
@@ -19,8 +19,11 @@ env:
   # Ensure that we build without warnings.
   - CARGO_FLAGS="--features 'strict'"
 
+before_script: (cargo install rustfmt || true)
```

Comment:
Aha, it's because `cache: cargo` saves the installed `rustfmt` so you don't need to install every time. Makes sense.